data/presets/highway/cycleway/traffic_island_shared.json

```
@@ -0,0 +1,36 @@
+{
+    "locationSet": {
+        "exclude": [
+            "fr",
+            "lt",
+            "pl",
+            "de",
+            "il",
+            "ps"
+        ]
+    },
+    "icon": "temaki-pedestrian_and_cyclist",
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "cycleway": "traffic_island"
```

This need to be more precise so we don't trigger on every `cw=traffic_island`, just like https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/highway/cycleway/bicycle_foot.json#L27-L30
```suggestion
        "cycleway": "traffic_island",
        "foot": "designated"
```